src/components/Footer/Footer.css

```diff
@@ -0,0 +1,40 @@
+.footer-container {
+    position: absolute;
+    bottom: 0;
+    /* margin-bottom: 0; */
+    height: 140px;
+    width: 100%;
+    /* background-color: #1f1f1f; */
+}
+
+.footer-button-container {
+    background-color: #333333;
+    height: 98px;
+    width: 100%;
+    text-align: center;    
+    /* margin: 0 auto; */
+    /* border-radius: 20px 20px 0 0; */
+}
+
+.footer-button-container>div {
+    display: inline-block;
+    position: relative;
+    bottom: -25px;
```

Comment:
I'm 100% sure you can avoid this.
Hint: One of the ways to do so is to use flex on `footer-button-container`